starter-code/package.json

```diff
@@ -9,8 +9,9 @@
   "author": "Code Fellows",
   "license": "ISC",
   "dependencies": {
-    "express": "^4.13.3",
     "body-parser": "^1.15.2",
+    "express": "^4.13.3",
+    "page": "^1.7.1",
```

Comment:
For now, npm and `package.json` should be reserved for server-side dependencies.

Eventually, though, this will change.